legacy/src/app/directives/pod-details/kvm_storage_dropdown.js

```diff
@@ -50,12 +50,20 @@ function KVMStorageDropdownController($scope, $filter) {
   $scope.poolOverCapacity = storage => {
     const { compose, pod } = $scope;
 
-    if (compose && compose.obj && pod && pod.storage_pools) {
+    if (
+      compose &&
+      compose.obj &&
+      compose.obj.requests &&
+      pod &&
+      pod.storage_pools &&
+      storage &&
+      storage.pool
+    ) {
```

Comment:
Couldn't that just be `if (compose?.obj?.requests && pod?.storage_pools && storage?.pool) {`?

Reply:
Oh yeah, you're right! This is on the 2.8 branch - I thought that babel wasn't yet transpiling that, but I guess it is.